Canvas.xcworkspace/contents.xcworkspacedata

```diff
@@ -16,9 +16,13 @@
    <FileRef
       location = "group:Pods/Pods.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:rn">
-   </FileRef>
+   <Group
+      location = "group:rn"
+      name = "rn">
+      <FileRef
+         location = "group:Teacher">
+      </FileRef>
+   </Group>
```

Comment:
This allowed me to set the tab width for the `rn` Group, and still have a FileRef for `Teacher`, so it just uses the file system instead of manually tracking the structure.